### PR TITLE
Add a module settings for show icon image of the items  on inventory version 2

### PR DESCRIPTION
**Pull request by p4535992**
OK , now i pretty confident, but i still don't like this part of the code:
```
      // Setting - Show icon on inventory list
      const settingsShowIconsOnInventoryList = game.settings.get(MODULE_ID, MySettings.showIconsOnInventoryList);
      sheetData?.inventory.forEach((itemType) => {
        itemType.items.forEach((item) => {
          if (!item.data.settings) {
            item.data.settings = {};
          }
          item.data.settings.showIconsOnInventoryList = settingsShowIconsOnInventoryList;
        });
      });
```
i don't like because i go to set the settings for each item, if you can suggest a workaround for using some global variable it will be perfect. 
Sincerely.

### CODE REVIEW

**Comment by akrigline** [comment]
Since this is a user setting, we should be able to put a field on `sheetData` based on that user's setting.

Inside `CompactBeyond5eSheet.getData()`
```
try {
  if (game.settings.get(MODULE_ID, MySettings.showIconsOnInventoryList)) {
    sheetData.showInventoryIcons = true;
  }
} catch (e) {
  log(true, 'error trying to set showInventoryImages');
}

// return sheetData;
```

Inside `actor-inventory.hbs`
```
{{#if "../showInventoryImages" }}
  // image code
{{/if}}
```
`../` goes up one 'context' within the Handlebars data. You can see this same pattern used for `../isCharacter` inside the inventory file.

**Comment by p4535992** [comment]
Wow, didn't know that trick. Ty very much for the tip. Updated the  pull.

File: src/foundryvtt-5eOGLCharacterSheet.ts [hunk 1]
```diff
@@ -3,6 +3,10 @@ import { registerSettings } from './module/settings.js';
 import { preloadTemplates } from './module/preloadTemplates.js';
 import { MODULE_ID, MySettings } from './constants.js';
 //@ts-ignore
+import { DND5E } from '../../systems/dnd5e/module/config.js';
```
Comment (akrigline) [inline src/foundryvtt-5eOGLCharacterSheet.ts]: I don't think these are used anymore.

**Review by akrigline — CHANGES_REQUESTED** [review]

**Comment by akrigline** [comment]
This looks great, thanks for the contribution!
I'll pull and test it tonight, merge after you make this tiny change and I test in 0.7.x.

**Review by akrigline — APPROVED** [review]
I'll pull this and do a quick test in 0.7.4 before merging, otherwise this looks done!

**Pull request merged**